website/docs/add-secure-apps/flows-stages/bindings/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Bindings
+---
+
+A binding is, simply put, a connection between two components (a flow, stage, policy, user, or group). The use of the binding adds additional functionality to one those existing components; for example, a policy binding can cause a new stage to be presented within a flow, for a specific user.
```

Comment:
Typo and suggestion: "The use of a binding can add additional functionality to an existing component..."